### ISSUE TITLE
GG Max - EXP - BUG - Armour is affected by Fall Damage

### ISSUE THREAD

**Issue by Tum1370** [TheGameCreators/GameGuruRepo#4502]
If you have the Armour stat set, If you jump off a building and take Fall Damage, The armour still reduces that damage.

Surely armour is only suppose to protect from bullets / magic damage ? And not fall damage !

**Comment by MonkeyFrogStudio** [comment]
Why wouldn't armor protect you from all sorts of damage? The point is that armor absorbs some of the impact, whether that's a blunt weapon, like a hammer, or a the blunt force trauma from a fall. And, yes, bullets, arrows, lasers, etc., too.

**Comment by MonkeyFrogStudio** [comment]
Rather than label this as a bug, how about a REQUEST for a feature to limit what armor protects the player from? Perhaps some armor only protects against magic, etc.

**Comment by Tum1370** [comment]
Am thinking more on the terms of Bullet Proof Vest and body armour, that stops bullets damage.
No game Armour protects from Fall damage.

**Comment by MonkeyFrogStudio** [comment]
And perhaps I am thinking more of medieval armor or space armor or ... or ... ??? And so what if no other game's armor protects you against fall damage. But here's the thing, using your Skyrim example, the armor in MAX only protects for so much damage, not full damage. So, if you fall off a cliff, and if that fall damage is massive, then it won't matter that the armor absorbed some of it ... your player would still be dead. 

So, again, instead of marking this as a bug (it's not, it's a design choice), put in a request that the end-user can decide which types of damage any armor can protect from and which it cannot. Then you can have armor work as you like it and someone else can have it as they like it.

**Comment by Kravenwolf** [comment]
It's a good idea, and I agree armor shouldn't affect fall damage. Realistically, armor wouldn't protect you from taking fall damage unless it's "magic armor" that stops both bullets and the effects of gravity. Not out of place in an RPG game maybe, but modern shooter games do tend to handle armor very differently. 

A tick box to either have the armor reduce or not reduce your fall damage would do the trick. :)

**Comment by Lemuripest93** [comment]
I don't think armor would protect against fall damage at all really, in fact - quite the opposite, in the Soulsborne franchise for instance, the heavier you are, the more fall damage you take, which makes logical sense - a heavy suit of armor is going to make you into a heavier object - thus increase fall velocity and the impact on fall. Though that's just how one developer decided to tackle it.

I think there's a case where armor could protect you from fall damage, maybe the armor you're using is magic or it's a weird sci-fi suit or something like MonkeyFrog suggested.

But ultimately i agree with Kravenwolf about the tick box, though i'd like to expand on this to have two tick boxes, one let's armor _**protect**_ you from falling, whereas one _**hurts**_ you _**more**_, you can only tick one box at a time, and ticking one cancels out the other naturally, ticking neither puts damage values at a regular state.

Personally i don't think we need to follow the laws of physics or realism, since not everyone wants to make the same game world, variety is the spice of life :)

**Comment by MonkeyFrogStudio** [comment]
I've worn medieval armor, even done a bit of fighting in it, too. So, yes, it certainly does make you heavier. But you normally don't just wear plates of metal or ring. You have plenty of padding between you and that. There's good reason for that. Remember, a piece of plate armor, like a breastplate (for example), is meant to take a hard blow from a war hammer, an axe, or a sword (and more). Have you ever seen someone measure the amount of pressure, the applied force that something like a war hammer can deliver? The padding between that plate and yourself absorbs a good bit of this, too, so even if the plate is dented, some of that additional impact is absorbed. I would say that falling in armor would not increase damage, but would probably decrease it, at least to some degree, due to the padding worn and due to the amount of impact it can take. Of course, that's going to vary depending on the armor - chain is not very good against impact, while plate is, etc. But both are worn with padding beneath them.

In any case, having a check box so that the end-user could decide is best. Certainly better than just one way or the other, imo.

**Comment by fearlesswee** [comment]
Putting the "realism" arguments behind, I think regardless it just makes more sense from a gameplay perspective for armor to not protect from fall damage. Fall damage is meant to be a sort of "logical barrier" to keep your player in the desired play area, and to discourage the player from taking illogical paths (such as jumping off of a ledge instead of taking the stairs down, but without outright removing the player's freedom to take those shorter routes at the cost of a bit of health.) If armor reduces fall damage, then as a level designer you're forced to design every fall with armor in mind, meaning dropoffs used to "softly" guide the player the right way would have to be comically extended, also making them all entirely lethal if the player happens to not have armor, which removes the interesting tradeoff decision of "Do I want to take this shorter route in exchange for some health?" 

Of course, since MAX is a game _engine/creator_ and not a _game_ by itself, this should be 100% upon the user to decide how they want to have it in their game, so I also push in favor of having a tick-box/check-box option for if armor should negate fall damage or not :)

**Comment by MonkeyFrogStudio** [comment]
@fearlesswee I agree. But games already have a mechanic that a shorter fall produces less damage than a longer fall. Even in MAX. All armor would do is absorb a bit of that (if the end-user wants it to, that is). But, yes, options are always the best, imo. :)

**Comment by fearlesswee** [comment]
@MonkeyFrogStudio I know fall damage scales with distance (it would be really strange if it didn't...imagine falling off a 5-ft ledge instakilling you lol). I was saying that if you wanted, for example, a scenario where the long way down is a winding path of stairs with enemies placed upon it, or a player could jump down a ledge to get down much faster whilst avoiding the enemies, having the fall being just far enough to leave the player barely alive (so that jumping down is an exchange between faster navigation and health, risk/reward decision), you have to take into account the armor you have in your game might allow them to easily make that jump and lose relatively little health, removing the risk/reward element from it. This, or you extend the fall so that _with_ armor the fall is barely survivable, but that runs into the issue of removing the option for players who didn't find armor as it's no longer survivable _without_ armor.

While one developer might want "armor negates fall damage risk/reward" or "you cannot survive most falls without armor" for their game, I feel that fall damage is too massively a useful tool for levels as a "soft barrier" (barriers that discourage certain actions/behaviors without _completely_ blocking it so that a player still has some freedom within the level, thus making the game feel a bit less "on rails") to be tied to the armor mechanic in any capacity, imo.

We do agree this should be toggleable by the developer anyhow, so this is all just discussion/thoughts on the topic and not any kind of debate LOL!
